sharktank/tests/models/llama4/llama4_test.py

```diff
@@ -28,7 +28,7 @@ def setUp(self):
 
     @pytest.mark.xfail(
         is_mi300x,
-        raises=TypeError,
+        raises=(TypeError, AssertionError),
```

Comment:
Why two different errors?

Reply:
If it's an assertion error, leave only that error there to ensure that the reason for the failure doesn't switch for an unrelated reason.